pages.de/common/emacs.md

```diff
@@ -0,0 +1,24 @@
+# emacs
+
+> Der erweiterbare, veränderbare, selbst-dokumentierende Echtzeit Text Editor.
+> Weitere information: <https://www.gnu.org/software/emacs>.
+
+- Starten von Emacs in der Konsole (ohne X-Fenster):
+
+`emacs -nw`
+
+- Öffnen einer Datei:
+
+`emacs {{path/to/file}}`
+
+- Tastenkombination zum Speichern einer Datei:
+
+`Ctrl + X, Ctrl + S`
+
+- Tastenkombination zum Beenden von Emacs:
+
+`Ctrl + X, Ctrl + C`
+
+- Öffnen einer Datei in einer bestimmten Zeile:
+
+`emacs +{{line_number}} {{path/to/file}}`
```

Comment:
Please translate the arguments for get this easier for non-English speakers, like `line_number` and `path/to/file` in both pages.